master/getting-started/kubernetes/upgrade/migrate.md

```diff
@@ -1,35 +1,52 @@
 ---
-title: Migrating data from etcdv2 to etcdv3
+title: Migrating Calico data
 no_canonical: true
 ---
 
 > **Important**: Once you begin the migration, stop using `calicoctl` or 
-> otherwise modifying the etcdv2 datastore. Such changes are unlikely to 
-> be migrated to the new datastore.
+> otherwise modifying the etcdv2 datastore. Any changes to etcdv2
+> data are unlikely to be migrated to the new datastore.
 {: .alert .alert-danger}
 
-1. Run `calico-upgrade start` to start the v1 to v3 data migration. This begins 
-   an interactive session. While existing connectivity will continue as before,
-   you cannot add any new endpoints until the migration and upgrade complete.
+
+1. To begin an interactive data migration session, use the `start` command. While 
+   existing connectivity will continue as before, you cannot add any new endpoints 
+   until the migration and upgrade complete.
+
+   **Syntax**
+   ```
+   calico-upgrade[-darwin-amd64|-windows-amd64.exe] start [--apiconfigv1 path/file] [--apiconfigv3 path/file]
+   ```
+   
+   **Reference**
+   
+   | Flag | Discussion 
+   | ---- | ---------- 
+   | <code>&#8209;&#8209;apiconfigv1</code> | By default, `calico-upgrade` looks for the etcdv2 configuration file at `/etc/calico/apiconfigv1.cfg`. If you have a configuration file in a different place or if it has a different name, include the `--apiconfigv1` flag and specify the name and location of the file. If you are using environment variables, you don't need this flag.
+   | <code>&#8209;&#8209;apiconfigv3</code> | By default, `calico-upgrade` looks for the etcdv3 configuration file at `/etc/calico/apiconfigv3.cfg`. If you have a configuration file in a different place or if it has a different name, include the `--apiconfigv3` flag and specify the name and location of the file. If you are using environment variables, you don't need this flag.
+   
+   **Example**
+   ```
+   calico-upgrade-darwin-amd64 start --apiconfigv1 etcdv2.yaml --apiconfigv3 etcdv3.yaml
+   ```
+   > **Note**: After this is started the {{site.noderunning}} pods may be restarted 
+   > by Kubernetes and enter the `CrashLoopBackOff` state. This is expected behavior. 
```

Comment:
It's not expected that they will enter `CrashLoopBackOff`, I don't think.

Reply:
I will check with @robbrockbank but he indicated that it was....

Reply:
We have seen this in testing.

Reply:
Other people have also indicated that it is expected for some time...but then it should resolve at the very end. @heschlie

Reply:
I wouldn't say expected, just that we had seen it, and you could expedite resolving it by kicking the pod manually, AFAIK we don't want the users to see it do that, and our liveness fixes were aiming to eliminate one big cause of it.

Reply:
Yeah, just because we've seen it crash in testing doesn't mean that's the expected behavior.... :p

I argue this isn't expected behavior and if we see it happen then it is a bug that should be fixed.